### PR TITLE
Announcements: Fix /announcement htmlcreate

### PR DESCRIPTION
**Pull request by Distrib-ps**
PR for https://www.smogon.com/forums/threads/bug-reports-v4-read-original-post-before-posting.3663703/post-8817595

### CODE REVIEW

File: server/chat-plugins/announcements.ts [hunk 1]
```diff
@@ -90,7 +90,7 @@ export const commands: ChatCommands = {
 				return this.errorReply(this.tr`There is already a poll or announcement in progress in this room.`);
 			}
 
-			const source = supportHTML ? this.checkHTML(target) : Chat.formatText(target, true);
```
Comment (mia-pi-git) [inline server/chat-plugins/announcements.ts:93]: This is removing all server side HTML sanitizing.

File: server/chat-plugins/announcements.ts [hunk 2]
```diff
@@ -90,7 +90,7 @@ export const commands: ChatCommands = {
...
+			const source = supportHTML ? Chat.collapseLineBreaksHTML(target) : Chat.formatText(target, true);
```
Comment (mia-pi-git) [inline server/chat-plugins/announcements.ts:93]:
```suggestion
			const source = supportHTML ? this.checkHTML(Chat.collapseLineBreaksHTML(target)) : Chat.formatText(target, true);
```

File: server/chat-plugins/announcements.ts (hunk 1 above)
Comment (Distrib-ps) [reply server/chat-plugins/announcements.ts:93]: Effctively, I have badly copied o/
thx

**Review by AnnikaCodes — APPROVED** [review]
Sure, this looks like a good change.

**Pull request merged**